global.json

```diff
@@ -0,0 +1,5 @@
+{
+    "sdk": {
+      "version": "2.2.300"
```

Comment:
This might break with the ASP.NET Core 3.x PR. You should just install a 3.0 SDK and build 2.1/2.2/3.0 with it.

Reply:
Tests failed when running under the 3.0 SDK...

Actually that might have been unrelated. Let me try again

Reply:
Ended up installing both SDKs.